src/core/TooltipTableContent/index.tsx

```diff
@@ -23,52 +19,27 @@ export { TooltipTableContentProps };
 const TooltipTableContent = (props: TooltipTableContentProps): JSX.Element => {
   const { alert, data, ...rest } = props;
 
-  const theme = useTheme();
-
-  const extraProps = {
-    /* stylelint-disable property-no-unknown -- false positive */
-    theme,
-    /* stylelint-enable property-no-unknown -- false positive */
-  };
-
-  const alertClassName = alertCss(extraProps);
```

Comment:
instead of generating these CSS classnames, we can generate styled components directly!

E.g.,

`alertCss(extraProps)` -> `Alert` component

```ts
export const Alert = styled.div`
  ${fontBodyXxs}
`;
```

And `rowLabelCss` -> `<RowLabel />`

```ts
export const RowLabel = styled(TableCell)`
  ${fontHeaderXs}
  ${disabledCss}

  padding: 0;
  width: 50%;
`;
```